### ISSUE TITLE
[Bug] upload and upload_multiple fields don't work as subfields inside relationship

### ISSUE THREAD

**Issue by tabacitu** [Laravel-Backpack/CRUD#4141]
# Bug report

### What I did

Tried to use `upload` and `upload_multiple` inside Cave and Story.

### What I expected to happen

Work.

### What happened

Uploads do not get saved, do not get shown.

### What I've already tried to fix it

Nothing

### Is it a bug in the latest version of Backpack?

v4.2 / v5 - whatever you call it

**Comment by tabacitu** [comment]
Alternative - create a field that works similarly (uploads without elFinder), but does work as subfield. Eg. `dropzone` or `media`

**Comment by mobielbekeken-jan** [comment]
maybe a good temporary solution to just show the images in the subfield and a link to the crud for the related model? Like  type: upload_multiple_view_only 

That would work for me, the user can see the related items with images and if it needs to be edited they can do that in another view.

**Comment by tabacitu** [comment]
I'm moving this to v5.2 but I don't really think we should do this. Rather than bending the `upload` and `upload_multiple` fields to work like this (which I expect will be very complicated), I think it's smarter to just create an alternative field.

> maybe a good temporary solution to just show the images in the subfield and a link to the crud for the related model? Like type: upload_multiple_view_only

Hmm... that would work in some use cases, indeed... but it wouldn't actually be a FIELD then, it would be a COLUMN (read-only)... so I don't think the use case is broad enough to warrant building it.

**Comment by tabacitu** [comment]
Yup - I agree with past-me. Let's do a different field for this. Probably https://github.com/Laravel-Backpack/ideas/issues/20 or https://github.com/Laravel-Backpack/ideas/issues/28 or both. KISS.
